www/form/app-form.less

```diff
@@ -605,15 +605,20 @@
                         }
                         .cp-form-block-question-text {
                             flex: 1;
+                            word-wrap: break-word;
+                            overflow: hidden;
                         }
                         .cp-form-required-tag {
                             padding: 5px;
                             margin-top: -10px;
                             margin-right: -10px;
                             &.cp-is-empty {
+                                max-height: fit-content;
                                 padding: 3px;
-                                border: 2px solid @cryptpad_color_red;
-                                color: @cp_form-invalid;
+                                border: 2px solid @cp_alerts-danger-fg;
+                                border-radius: @variables_radius;
```

Comment:
Border radius should be none except at the top-right where the question container is rounded. So `0 @variables_radius 0 0`